contracts/HATVaults.sol

```diff
@@ -112,8 +112,8 @@ contract  HATVaults is Governable, HATMaster {
         _;
     }
 
-    modifier noPendingApproval(uint256 _pid) {
-        require(pendingApprovals[_pid].beneficiary == address(0), "HVE02");
+    modifier noSubmittedClaims(uint256 _pid) {
```

Comment:
lets make it `noActiveClaims`

Reply:
i would not add another word into this terminology actually..